.storybook/symbol-doc-loader.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
-const {createDocProgram} = require('../modules/docs/docgen/createDocProgram');
```

Comment:
why was this removed?

Reply:
Just making a note here, I think it's because Nicholas is creating the program now in `main.js`. Could be wrong though

Reply:
Yes. The program is created in `main.js`. I think eventually it makes sense to have a Webpack plugin that manages the program instead though.